src/platform-integration/web/faq.md

```diff
@@ -100,17 +100,65 @@ although no such support is built in.
 
 ### How do I embed a Flutter web app in a web page?
 
-You can embed a Flutter web app,
-as you would embed other content,
-in an [`iframe`][] tag of an HTML file.
-In the following example, replace "URL"
-with the location of your hosted HTML page:
+You have two embed options: embed in an `iframe` and embed in a `div` container.
+
+#### Use an `iframe`
+
+To embed a Flutter web app in an [`iframe`][] tag of your HTML file,
+add the tag and link to the HTML file that contains your Flutter app.
+
+In the following example, replace `URL` with the location of your HTML page.
 
 ```html
 <iframe src="URL"></iframe>
 ```
 
-If you encounter problems, please [file an issue][].
+#### Use a `div` container
+
+To embed your Flutter app using a `<div>` element as a container,
+you need to create the element and add some JavaScript in your HTML page.
+
+In the HTML file you want to embed your Flutter-generated JavaScript,
+make the following changes.
+
+1. Create a `<div>` tag with a unique `id` attribute.
+   You will reference this `id` in a later step. 
+
+   ```html
+   <div id="flutter-container"></div>
+   ```
+
+1. To initialize your Flutter app, add the following JavaScript code.
+   Place this code in a `<script>` tag after the `<script>` tags from the previous step.
+   To specify the entry point of your Flutter app, use the `flutter.runApp` function.
+   Use the `id` you set in the previous step of this procedure.
+
+   ```html
+   <script>
+    flutter.runApp(
+      {
+        element: document.getElementById('flutter-container'),
+        initialRoute: '/',
+        routes: {
+          '/': (_) => MyApp(),
+        },
+      },
+    );
```

Comment:
Is this generated by ChatGPT? This has **nothing** to do with how a Flutter app is initialized from Javascript.